src/usr/local/www/classes/Form/Checkbox.class.php

```diff
@@ -56,9 +56,9 @@ protected function _getInput()
 	{
 		$input = parent::_getInput();
 
-		if (empty($this->_description))
-			return '<label class="chkboxlbl">'. $input .'</label>';
+		if (!empty($this->_description) || is_numeric($this->_description))
```

Comment:
I switched this logic around to use the exact same boolean construction as in other Form class files. That should make it easier for future readers to understand, compare and debug.